src/main/java/com/github/isopropylcyanide/jdbiunitofwork/core/LinkedRequestScopedJdbiHandleManager.java

```diff
@@ -85,7 +84,7 @@ public void clear() {
 
     @Override
     public ThreadFactory createThreadFactory() {
-        String threadName = String.format(NAME_FORMAT, getConversationId());
+        String threadName = String.format("[%s]-%%d", getConversationId());
```

Comment:
For readability, the format should be in line instead of in a constant that makes it harder to compare the params passed